src/viam/ur/module/ur_arm.cpp

```diff
@@ -372,6 +374,11 @@ void URArm::configure_(const std::unique_lock<std::shared_mutex>& lock, const De
     current_state_->host = find_config_attribute<std::string>(cfg, "host");
     current_state_->speed.store(degrees_to_radians(find_config_attribute<double>(cfg, "speed_degs_per_sec")));
     current_state_->acceleration.store(degrees_to_radians(find_config_attribute<double>(cfg, "acceleration_degs_per_sec2")));
+    try {
+        reject_move_request_threshold_deg.emplace(find_config_attribute<double>(cfg, "reject_move_request_threshold_deg"));
```

Comment:
validate non negative & non zero. if we want to do this in a sepatate pr since we don't validate the other config fields atm thats fine with me